source/inputCore.py

```diff
@@ -452,6 +452,22 @@ def __eq__(self, other: Any) -> bool:
 		return NotImplemented
 
 
+decide_handleRawKey = extensionPoints.Decider()
```

Comment:
Would it not be better to indicate the signature of the handler in the docstring and remove each individual param type description?

Reply:
Could you provide an example of what that would look like?